--- /home/kaffein/.hermes/cache/scratch/k47/xnu-xnu-12377.41.6/osfmk/vm/vm_map_xnu.h	2025-12-01 20:55:51.000000000 +0100
+++ /home/kaffein/.hermes/cache/scratch/k47/xnu-xnu-12377.61.12/osfmk/vm/vm_map_xnu.h	2026-01-07 23:53:10.000000000 +0100
@@ -476,7 +476,7 @@
 	/* reserved */ res0:1,
 #endif /* HAS_MTE || HAS_MTE_EMULATION_SHIMS */
 	/* reserved  */pad:6;
-	unsigned int            timestamp;          /* Version number */
+	uint64_t timestamp;          /* Version number */
 	/*
 	 * Weak reference to the task that owns this map. This will be NULL if the
 	 * map has terminated, so you must have a task reference to be able to safely
@@ -564,7 +564,7 @@
  *		Just a timestamp for the main map.
  */
 typedef struct vm_map_version {
-	unsigned int    main_timestamp;
+	uint64_t    main_timestamp;
 } vm_map_version_t;
 
 /*
